pkg/client/types.go

```diff
@@ -66,4 +66,7 @@ type APIResponse struct {
 	ErrorType ErrorType `json:"errorType"`
 	// Error is the error message, if this is an error response.
 	Error string `json:"error"`
+
+	// StatusCode is the status code returned by the server
+	StatusCode int `json:"statusCode"`
```

Comment:
add statusCode to apiresponse for information propagation. When parsing the json this should always be empty, but we manually set it before returning the object

Reply:
ditto: Do we need this? If there is no error returned with this, we should assume it's a 200 OK.